packages/paste-website/src/pages/experiences/console/index.mdx

```diff
@@ -0,0 +1,379 @@
+  {
+    mdx(frontmatter: {slug: {eq: "/experiences/console/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<content>
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  description={props.pageContext.frontmatter.description}
+  categoryRoute={SidebarCategoryRoutes.EXPERIENCES}
+/>
+
+</content>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Base principles
+
+- Do not duplicate Console content in different places
+- Focus on the user journey, not just your product or feature
+- Rely on existing patterns before creating a new one
+- Use Paste with the Default theme for all new content
+- Console navigation is optimized for findability, not promoting products or features
+
+## Develop vs. Monitor
+
+In our research, we found that a majority of users expect content to be organized by
+the phases of their development lifecycle, not by product vertical.
+A user's journey in building an application on Twilio starts with **Developing**,
+then moves into **Monitoring** after they launch their application.
+
+<Box marginBottom="space40">
+<Table tableLayout="fixed">
+<TBody>
+<Tr>
+<Td width="50%">
+<strong>
+Develop
+</strong>
+
+The customer is getting started, learning about how Twilio works, testing out Twilio product(s), prototyping a POC,
+buying and configuring numbers, setting up services, building their application, and integrating it into their own
+system.
+
+Primary Personas: Developers, Business and Product leaders
+
+</Td>
+<Td width="50%">
+<strong>
+Monitor
+</strong>
+
+The customer has "launched" their application into production and needs to monitor it's activity,
+coming back to Console to troubleshoot, debug, review logs and insights.
+
+Primary Personas: Customer support, Developers, and Product Managers
+
+</Td>
+</Tr>
+
+</TBody>
+</Table>
+</Box>
+<br />
+<br />
+
+When building your product, think about the journey your users will take.
+Within **Develop**, a user should be able to accomplish all tasks related to learning about the product,
+trying it out, developing and managing services, and applying product settings.
+Within **Monitor**, a user should be able to find all content related to monitoring their application,
+such as logs and insights.
+
+On a rare occasion, there are tasks that span between Develop and Monitor.
+For example, a user may learn that calls are not connecting in India through Voice Insights (in **Monitor**),
+and that they need to turn on India in their Voice Geo-permissions (in **Develop**). In cases like these,
+good interlinking between the relevant pages or contextual quick actions will provide a streamlined
+user experience. In addition, the tabs will hold the menu states so that less clicks are required to drill
+in and between menus under each tab.
+
+![](../../../assets/images/experiences/console/develop-or-monitor.png)
```

Comment:
question: should we give all these images alt text or is the image caption below enough?

Reply:
Image captions are fine! I added comments below to the ones that I think could be more informative.